src/core/etl/tests/Flow/ETL/Tests/Benchmark/Row/Reference/Expression/AddJsonBench.php

```diff
@@ -0,0 +1,43 @@
+<?php declare(strict_types=1);
+
+namespace Flow\ETL\Tests\Benchmark\Row\Reference\Expression;
+
+use function Flow\ETL\DSL\lit;
+use function Flow\ETL\DSL\ref;
+use Flow\ETL\DSL\From;
+use Flow\ETL\DSL\To;
+use Flow\ETL\Flow;
+use Flow\ETL\Memory\ArrayMemory;
+use PhpBench\Attributes\Iterations;
+use PhpBench\Attributes\Revs;
+
+final class AddJsonBench
```

Comment:
Looking at the benchmarks I would probably rename it to `ArrayMergeExpressionBench` or something similar, AddJsonBench seems to not describe properly what is really tested here (or maybe I'm missing something)

Reply:
This was first test, I replaced it with benchmark based on one of the examples.